lib/reek/context/module_context.rb

```diff
@@ -7,6 +9,19 @@ module Context
     # A context wrapper for any module found in a syntax tree.
     #
     class ModuleContext < CodeContext
+      include Enumerable
```

Comment:
Fixed with some inspiration from http://blog.arkency.com/2014/01/ruby-to-enum-for-enumerator/